with-better-auth/app/(auth)/sign-up.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import { TextInput, Button, ScrollView, Text, StyleSheet } from "react-native";
+import { authClient } from "@/lib/auth-client";
+
+export default function SignUp() {
+  const [email, setEmail] = useState("");
+  const [name, setName] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSignUp = async () => {
+    await authClient.signUp.email({
+      email,
+      password,
+      name,
+    });
```

Comment:
I immidiately ran into a "password-too-short"-error. Would it make sense to handle errors in this example?

e.g.
```suggestion
const signUpResponse = await authClient.signUp.email({
      email,
      password,
      name,
});

if (signUpResponse.error) {
  Alert.alert("Error", signUpResponse.error.message);
  return;
}
```